zwave_js_server/model/node/__init__.py

```diff
@@ -466,8 +468,20 @@ async def async_send_command(
         if wait_for_result or (
             wait_for_result is None and self.status != NodeStatus.ASLEEP
```

Comment:
Why aren't we checking for dead status too here?

Reply:
good question? no idea...